src/Halogen/ECharts.purs

```diff
@@ -78,19 +80,20 @@ echarts
   . (MonadAff (EChartsEffects eff) g)
   ⇒ H.Component HH.HTML EChartsQuery (Dimensions /\ Unit) EChartsMessage g
 echarts =
```

Comment:
we could add `echarts''` which takes `theme` but not `receiver` or export this default receiver so one could pass it alongside theme to `echats'`